data-prepper-plugins/http-source/src/main/java/com/amazon/dataprepper/plugins/source/loghttp/HTTPSource.java

```diff
@@ -44,8 +51,19 @@ public void start(Buffer<Record<String>> buffer) {
         }
         if (server == null) {
             final ServerBuilder sb = Server.builder();
-            // TODO: allow tls/ssl
-            sb.http(sourceConfig.getPort());
+            if (sourceConfig.isSsl()) {
```

Comment:
Nit: do we want to have `isSsl` or `isSSL` given that SSL is an acronym and not a proper word?

Reply:
I think isSSL makes more sense. This getter is autogenerated.

Reply:
I think we should stick with `isSsl`. Making initials lowercase is a common convention. For example, `java.net.http.HttpRequest`.